Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM adoptopenjdk/maven-openjdk11:latest
+
+RUN apt-get update
+RUN apt-get -yq clean
+RUN apt-get --yes --force-yes install firefox
+
+RUN groupadd -g 985 rhsvc && \
+    useradd -m -r -u 985 -g rhsvc rhsvc
+
+ENV M2_HOME=/home/rhsvc/.m2
+RUN mkdir -p /home/rhsvc/.m2/repository
+COPY . /home/rhsvc/census-cc-cucumber
```

Comment:
I have replaced census-cc-cucumner with census-field-service-cucumber wherever it occurs now. I have also added the relevant apt-get commands for getting the latest stable version of Firefox (something I found out how to do for census-rh-cucumber today). However, I will also need to update the PATH variable when I have found out what folder Firefox gets installed into.

Reply:
I don't need to update the PATH variable after all. However, I have now added another line, which creates a symbolic link, so that it should work the same way as census-rh-cucumber does in Concourse (in terms of how it installs Firefox and runs it)